addons/mail/static/src/js/utils.js

```diff
@@ -84,6 +85,38 @@ function parseEmail(text) {
     }
 }
 
+/**
+ * Returns an HTML conversion of a content.
+ *
+ * @param {string} content
+ * @returns {string}
+ */
+function getHtmlContent(content) {
+    //Removing unwanted extra spaces from message
+    let value = _.escape(content).trim();
+    value = value.replace(/(\r|\n){2,}/g, '<br/><br/>');
+    value = value.replace(/(\r|\n)/g, '<br/>');
+
+    // prevent html space collapsing
+    value = value.replace(/ /g, '&nbsp;').replace(/([^>])&nbsp;([^<])/g, '$1 $2');
+    return value;
+}
+
+/**
+ * Replaces the content between selectionStart and selectionEnd positions by an other content.
+ *
+ * @param {string} initialContent
+ * @param {string} addedContent
+ * @param {integer} selectionStart
+ * @param {integer} selectionEnd
+ * @returns {string}
+ */
+function insertTextContent(initialContent, addedContent, selectionStart, selectionEnd) {
+    let partA = initialContent.slice(0, selectionStart);
+    let partB = initialContent.slice(selectionEnd, initialContent.length);
+    return partA + addedContent + partB;
+}
```

Comment:
The generic behavior of this function may only be used once at the moment but it could be used elsewhere (the purpose of a function). Also 4 params are not that much.

Reply:
I wouldn't split to a utility function yet

Reply:
Done (I find it uggly)